src/view/dataElements/xdmObject.jsx

```diff
@@ -226,39 +227,22 @@ const XdmObject = forwardRef(({ initInfo, resetForm }, ref) => {
       {editorAreaContent}
     </div>
   );
-});
+};
 
 XdmObject.propTypes = {
-  initInfo: PropTypes.object.isRequired,
-  resetForm: PropTypes.func.isRequired
+  initInfo: PropTypes.object,
+  formikProps: PropTypes.object,
+  registerImperativeFormApi: PropTypes.func
 };
 
 const XdmExtensionView = () => {
-  const childRef = useRef();
-
   return (
     <ExtensionView
       render={() => {
         return (
           <ExtensionViewForm
-            initialValues={{}}
-            getSettings={({ values }) => {
-              return childRef.current.getSettings({ values });
-            }}
-            validateFormikState={({ values }) => {
-              return childRef.current.validateFormikState({ values });
-            }}
-            validateNonFormikState={() => {
-              return childRef.current.validateNonFormikState();
-            }}
-            render={({ initInfo, resetForm }) => {
-              return (
-                <XdmObject
-                  ref={childRef}
-                  initInfo={initInfo}
-                  resetForm={resetForm}
-                />
-              );
+            render={props => {
+              return <XdmObject {...props} />;
```

Comment:
The changes to XdmObject were just to accommodate the refactored ExtensionViewForm.